pyfarm/agent/service.py

```diff
@@ -118,6 +115,60 @@ def agents_endpoint(cls):
         """
         return config["master_api"] + "/agents/"
 
+    @inlineCallbacks
+    def schedule_call(self, delay, function, *args, **kwargs):
+        """
+        Schedules ``function`` to be run after ``delay`` has elapsed.
+
+        :param int delay:
+            Number of seconds to delay calling ``function``
+
+        :param function:
+            A callable function to run
+
+        :param args:
+            Arguments to pass into ``function``
+
+        :param kwargs:
+            Keywords to pass into ``function``
```

Comment:
These two arguments should be differently named. As it stands now, scheduled functions cannot have parameters with the same name as any parameter of ```schedule_call```.